src/app/page/categoryInfo/course/course.component.html

```diff
@@ -9,10 +9,24 @@
         <star-rating-comp [starType]="'svg'" [showHalfStars]="true"
                           [rating]="course.rating" (onClick)="onRatingClick(course, $event)"></star-rating-comp>
       </td>
-      <ng-template #unAuthorized>
+            <ng-template #unAuthorized>
         <star-rating-comp [starType]="'svg'" [showHalfStars]="true"
                           [rating]="course.rating" [readOnly]="true"></star-rating-comp>
       </ng-template>
+      <td class="alignCenter">
+        <div *ngIf="!course.status">
+          <div (click)="addCourseToUser(course.courseId)" *ngIf="isAuthorized()"
+               class="button addDeckToUserButtonActive"
+               title="Subscribe to the course"></div>
+        </div>
+        <div *ngIf="course.status">
+          <div (click)="addCourseToUser(course.courseId)" *ngIf="isAuthorized()"
```

Comment:
Why method addCourseToUser deletes course?

Reply:
This method has been already added earlier, so I just used it. It adds the course to the user if this user doesn't have one. Otherwise it deletes it. And I think it's normal, but it'd be good to add this to documentation later.